src/app/components/play-round/play-round.component.html

```diff
@@ -10,7 +21,8 @@
   mat-raised-button
   class="interact-button"
   color="primary"
-  *ngIf="this.foundRoundMarker$ | async as marker" [disabled]="!marker.active"
+  *ngIf="this.foundRoundMarker$ | async as marker"
+  [disabled]="!marker.active || ((this.studentRoundService.currentBeePollen$ | async) >= 3 && marker.barcodeValue != 0)"
```

Comment:
Just one more, sorry!

```suggestion
  [disabled]="!marker.active || ((this.studentRoundService.currentBeePollen$ | async) >= 3 && !marker.isNest)"
```

Reply:
Oops, I didn't see that one. Good catch!